src/common/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, Input } from "../Button/";
+import { dictionary } from "../../strings";
+import { SearchPanelWrapper } from "./styled";
+import { SearchBarProps } from "./types";
+
+export const SearchBar = ({ handleSearch, searchValue, handleSearchInput }: SearchBarProps) => {
+
+  return (
+    <SearchPanelWrapper>
+      <Input placeholderText={dictionary.searchPlaceholder} value={searchValue} onChange={handleSearchInput} />
+      <Button buttonText={dictionary.buttonSearch} handleClick={() => handleSearch(searchValue.trim())} />
```

Comment:
Do you need to pass searchValue to the component and pass searchValue to handleSearch?
Think about moving the logic into a component